component.json

```diff
@@ -37,6 +37,19 @@
       "metadata": {
         "out": {}
       }
+    },
+    "write_attachment": {
+      "main": "./lib/actions/write.js",
+      "title": "Write CSV attachment",
+      "fields": {
+        "reader": {
```

Comment:
Is't it confusing, that `write` action has `reader` field in config? How about a name which contains `write`?